agent/kratos/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+kratos migrate sql sqlite:///app/data/kratos.sqlite?_fk=true --yes # this initializes the db
+kratos serve -c /etc/config/kratos/kratos.yml --dev --watch-courier # start kratos
```

Comment:
--dev is for development?

Reply:
good catch, not needed anymore 
removed it